src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -298,7 +300,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
       isPublishedEmailEnabled: feedbackSession.isPublishedEmailEnabled,
 
       isSaving: false,
-      isEditable: false,
+      isEditable: isEditable,
```

Comment:
Lint check seems to be failing because of this

```suggestion
      isEditable,
```